runner/execer/os/os_execer.go

```diff
@@ -14,9 +16,13 @@ func NewExecer() execer.Execer {
 type osExecer struct{}
 
 func (e *osExecer) Exec(command execer.Command) (result execer.Process, err error) {
+	if len(command.Argv) <= 0 {
+		return nil, errors.New("No command specified.")
```

Comment:
Should we return a specific error type so the caller can differentiate between bad request and something the execer couldn't do?